### ISSUE TITLE
[FEATURE REQUEST] Parametrize to check also closed PRs

### ISSUE THREAD

**Issue by gladykov** [agrc/reminder-action#10]
Great tool, but it checks only opened PRs:

https://github.com/agrc/reminder-action/blob/c9dca07e06ab280b813702662cce06acd0b47452/index.js#L39

Would it be possible to paramterize it, so it also iterate over closed PR's?

We often merge some code, and close, but we want to be reminded about some code change in one month, to revert it or recheck.

**Comment by kublaios** [comment]
I have the exact same use case, is this a won't fix?

**Comment by steveoh** [comment]
> I have the exact same use case, is this a won't fix?

No it's not a won't fix but it's not a feature we use so it will likely be a community contribution.

**Comment by kublaios** [comment]
I see. How large do you think the scope is? Would simply adding more states to the call suffice or it's more complicated than that?

**Comment by steveoh** [comment]
Yah that might be. But th op wants to be able to configure it so a bit of file reading would be required. I think the scope is a few hours.

**Comment by gladykov** [comment]
For now forked version which will read all closed PRs and issues by default: https://github.com/gladykov/reminder-action/commit/70cac9925c5583a56536ac4cfb4e375eab00766b

 
For us is not a problem, as we are not using issues at all. We also limited action to run only once per day, not to use too much resources.

**Comment by jd-solanki** [comment]
@steveoh I want to reopen the issue after a certain time and this action is a great candidate for usage but unfortunately it doesn't remind on closed issue 😞

**Comment by steveoh** [comment]
I'm going to close this as it doesn't seem like anyone is wanting to contribute this feature and we aren't going to add it. Basically it's a stale issue. Thanks everyone! We are still open to a pull request to add the feature.
